Dockerfile

```diff
@@ -27,7 +27,7 @@ ADD LICENSE /usr/share/doc/python-crane/
 ADD COPYRIGHT /usr/share/doc/python-crane/
 ADD README.rst /usr/share/doc/python-crane/
 
-RUN pip-python install /usr/local/src/crane/
+RUN pip install /usr/local/src/crane/
```

Comment:
The name appear to have changed back to ``pip`` at some point. The only place I can find mention of it is this issue from 2013: https://bugzilla.redhat.com/show_bug.cgi?id=958377

I don't know why that change wasn't present in epel7 when this dockerfile was created. Oh well.